docs/fr/4-Collaboration/Environnement-Analyse-Geospatiale.md

```diff
@@ -165,10 +167,13 @@ plt.show()
 ```
 
 **Exemple : librairie ipyleaflet**
-Dans cet exemple, nous utiliserons la librairie 'ipyleaflet' pour créer une carte interactive. Cette carte sera centrée autour de Toronto, ON. Les données utilisées seront décrites ci-dessous.
+Dans cet exemple, nous utiliserons la librairie `ipyleaflet` pour créer une carte interactive. Cette carte sera centrée autour de Toronto, ON. Les données utilisées seront décrites ci-dessous.
```

Comment:
When talking about a software library, is the translation "bibliothèque" or "librairie"?

Reply:
bibliotheque